presto-docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -0,0 +1,71 @@
+====================
+ClickHouse Connector
+====================
+
+The ClickHouse connector allows querying tables in an external
+`Yandex ClickHouse <https://clickhouse.tech/>`_ instance. This can be used to join data between different
+systems like ClickHouse and Hive, or between two different ClickHouse instances.
+
+Configuration
+-------------
+
+To configure the ClickHouse connector, create a catalog properties file
```

Comment:
Try "To configure the ClickHouse connector, create a catalog properties file ``etc/catalog/clickhouse.properties``. Replace the connection properties as appropriate for your setup:"